src/Language/PureScript/Pretty/Values.hs

```diff
@@ -134,7 +135,7 @@ prettyPrintDeclaration d (BindingGroupDeclaration ds) =
   where
   toDecl (nm, t, e) = ValueDeclaration nm t [] (Right e)
 prettyPrintDeclaration d (PositionedDeclaration _ _ decl) = prettyPrintDeclaration d decl
-prettyPrintDeclaration _ _ = internalError "Invalid argument to prettyPrintDeclaration"
+prettyPrintDeclaration _ x = text ("-- " ++ show x)
```

Comment:
I like the desugared ast to be pretty printed. But `prettyPrintDeclaration` only implements half of the declarations. So if it encounters an unimplemented declaration it just switches to `show` here.